### PR TITLE
[8.1] add grafana demo dashboard

### PR DESCRIPTION
**Pull request by EwoudK**
BEGINRELEASENOTES
*dashboards
NEW: added json export of Grafana demo dashboard

ENDRELEASENOTES

### CODE REVIEW

**Comment by chaen** [comment]
Of what is it the demo ? :-) 
Can't it fit  into the existing category ? (WMS, DataManagement, etc).

**Comment by EwoudK** [comment]
It is the dashboard I demo'd in the last BiLD meeting (09/03) that shows the features of grafana combined with the MySQL databases and ElasticSearch clusters from the DIRAC certification version. It does not really have a theme I would say and so I was not sure where to fit it in the existing categories but if you can point me to the correct one, I can change it.

**Comment by chaen** [comment]
Right. A note for ourself would be to start documenting all these dashboards...
In the meantime, could you please add a short `README.md` in which you explain roughly what the dashboard contains, as well as which data source  are needed ? 
Thanks !

File: dashboards/GrafanaDemo/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+## Data sources used
+- MySQL Accounting database for DIRAC certification
+- ElasticSearch cluster for DIRAC certification
+
+## What plots does the dashboard contain?
+- Production Jobs By Final Minor Status
+- User Jobs By Final Minor Status
+- Job CPU Efficiency By Job Type
+- User Jobs By Job Type
+- User Jobs By Final Minor Stattus
```
Comment (fstagni) [inline dashboards/GrafanaDemo/README.md]:
```suggestion
- User Jobs By Final Minor Status
```

File: dashboards/GrafanaDemo/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Grafana Demo
+
+## Data sources used
+- MySQL Accounting database for DIRAC certification
+- ElasticSearch cluster for DIRAC certification
+
+## What plots does the dashboard contain?
```
Comment (fstagni) [inline dashboards/GrafanaDemo/README.md:5]: Just specify which source (MySQL or ES) is used for each of these plots

File: dashboards/GrafanaDemo/README.md
Comment (fstagni) [inline dashboards/GrafanaDemo/README.md:1]: Can you write on which grafana dashboard has this been tested?

**Review by fstagni — APPROVED** [review]

**Pull request merged**